internal/model/table.go

```diff
@@ -231,6 +231,7 @@ func (t *Table) reconcile(ctx context.Context) error {
 	}
 	if err != nil {
 		log.Error().Err(err).Msg("Reconcile failed to list resource")
+		return err
```

Comment:
@vimalk78 Thank you! I see it now , I think we can let it go and check check that we indeed have rows before kicking off the renderers.

Reply:
did you want me to remove this return?

Reply:
removed return